CONTRIBUTING.md

```diff
@@ -111,13 +120,12 @@ If an issue is a `bug`, and it doesn't have a clear reproduction that you have p
 
 - `bug`s should be closed when the issue is fixed and merged to `master`
```

Comment:
I think merged myself. Otherwise we need another label `fixed` or the like and we mentally have to ignore `fixed` issues. I appreciate users might not find an issue that has been closed but not yet released however.